example/ruby.lua

```diff
@@ -0,0 +1,16 @@
+-- Enable rubocop formatting in ruby
+require('formatter').setup({
+  logging = false,
+  filetype = {
+    ruby = {
+      -- rubocop run on one file
+      function()
+        return {
+          exe = "rubocop", -- might prepend `bundle exec `
+          args = { '--auto-correct', '--stdin', '%:p', '2>/dev/null', '|', 'sed "1,/^====================$/d"' },
```

Comment:
```suggestion
          args = { '--auto-correct', '--stdin', '%:p', '2>/dev/null', '|', "awk 'f; /^====================$/{f=1}'"},
```
This will work even if the first line is all `======`

Reply:
I am probably not the best person to review awk codes, but as I understand it, your solution means: ignore from line number 1 to the delimiter and return everything else. What it does better than the original sed function is that because sed relies on the range to perform its operation, if the first line is also the delimiter it will delete everything onwards. Meanwhile, the aws script can work just fine. 

Once you confirm this is correct, I am happy to commit the suggestion.

Reply:
Yes, that is my understanding of the differences as well.

Reply:
Thank you so much for your suggestion